### ISSUE TITLE
Auto refresh browser window with folder contents

### ISSUE THREAD

**Issue by Hymenorebutia** [iterate-ch/cyberduck#14640]
Hi "Cyberduck",
As usual, I am much satisfied by Cyberduck… BUT…
I encounter always the same problem:
- I manage an e-library where I add/update/delete new items to the remote server, and Cyberduck is thus always running in the background on my computer. Some friends have access to the remote server, too.
- When I come back to (reactivate) the cyberduck page of my remote server (let us say 1 or 2 days later my last activation), I see the file list that I saw at my previous activation.
- If a friend modified the content of the remote server in the meantime, I DO NOT SEE THE MODIFICATION; I see the list of file as it was at my previous visit… I need to click the "refresh" icon top of the Cyberduck window to get the new file list…
- I often forget to refresh the page, and I desperately look for a file that is invisible although it is present!!!

Would it be possible to have an automatic refreshing of the transfer window when it is reactivated after an idling period in the background?
I hope that my question is not too much stupid and that this "enhancement request" is at the relevant location… within this "bug report" list…

**Comment by dkocher** [comment]
> Would it be possible to have an automatic refreshing of the transfer window when it is reactivated after an idling period in the background?

I assume you refer to the _Browser_ window.

**Comment by Hymenorebutia** [comment]
Yes

**Comment by Dahn-YoungDong** [comment]
yes please. Upvote this feature request- for automatic refreshing of browser content
